e2e/testsuite/tx.go

```diff
@@ -201,7 +243,39 @@ func (s *E2ETestSuite) waitForGovV1ProposalToPass(ctx context.Context, chain ibc
 
 // ExecuteAndPassGovV1Beta1Proposal submits the given v1beta1 governance proposal using the provided user and uses all validators to vote yes on the proposal.
 // It ensures the proposal successfully passes.
-func (s *E2ETestSuite) ExecuteAndPassGovV1Beta1Proposal(ctx context.Context, chain ibc.Chain, user ibc.Wallet, content govtypesv1beta1.Content) {
+func (s *E2ETestSuite) ExecuteAndPassGovV1Beta1Proposal(ctx context.Context, chainA ibc.Chain, user ibc.Wallet, content govtypesv1beta1.Content, chainB ibc.Chain) {
```

Comment:
I think chainB doesn't need to be an arg, this is just being added because of the `BroadcastMessages` method signature update right?